### ISSUE TITLE
Move 'Videos' section below 'Features' 

### ISSUE THREAD

**Issue by m-renaud** [haskell-infra/hl#245]
**Note:** For those new to this thread, please read [my comment](https://github.com/haskell-infra/hl/issues/245#issuecomment-448092444) below for a more fleshed out idea. 

The 'Videos' section seems to be given very high priority relative to other content on the home page. I can't think of a reason why as a newcomer would want to see a list of videos (one of which is called "Haskell is Not for Production..." :stuck_out_tongue_closed_eyes:) before seeing what the language features are or why they would want to use it. This is an even bigger issue on mobile where the videos take up more than an entire screen worth of real estate so you need to scroll down several pages before you get to the Features section.

I propose we either:

a) Move the videos further down the page (at least under 'Features').
b) Remove the videos from the page entirely (could be moved to a new "Learn" page which would be merged with "Documentation")

Implementing the first should be as simple as swapping [these two lines](https://github.com/haskell-infra/hl/blob/843e93c7fc4e46e8bb875c4acb7f541c43fcf688/src/HL/View/Home.hs#L26-L27). I can send a PR for this as well if folks agree.

/cc @gbaz

**Comment by gbaz** [comment]
I tend to disagree. I think the videos are a really nice element of the page -- people like them, as illustrated by the healthy discussion when we revamped the video selection. (https://www.reddit.com/r/haskell/comments/6hpfr9/what_are_your_favorite_introductory_haskell_videos/  and https://github.com/haskell-infra/hl/pull/229). They illustrate we have a vibrant language community even if people don't watch them all at once. And further, they break up the text-heavy visual monotony that would occur if we had "features" first, so make the page as a whole more engaging. I just checked on mobile and they're not too bad there either. I don't think there's enough benefit to this change to outweigh how much less engaging the page would be as a whole.
